### ISSUE TITLE
Rename "Publish Later" in new blog post page to "Publish Later At"

### ISSUE THREAD

**Issue by VahidNaderi** [OrchardCMS/Orchard#7056]
I suggest replacing the text and order of elements like this
![orchardissue](https://cloud.githubusercontent.com/assets/3517756/17103031/7ddb8c94-5291-11e6-99ca-13f7a6a99f82.jpg)

It makes more sense and prevents user's first failed try frustration.

**Comment by sfmskywalker** [comment]
I agree, much clearer.

**Comment by VahidNaderi** [comment]
@sfmskywalker do you want me to make a pull request for it?

**Comment by sfmskywalker** [comment]
I do, but we should wait until we triaged this suggestion with the rest. We do triage meetings every Thursday, hopefully we will get to this one. Thanks

**Comment by sebastienros** [comment]
I have another suggestion, to replace the Publish Later button by a checkbox, which would show the DateTime editor. And then use the standard Publish button.

**Comment by Skrypt** [comment]
The "publish later" button should open a modal in which we could then pick a date. That modal would have a cancel and "publish later" button ... That way we prevent users from not picking up a date before pushing the "publish later" button.

**Comment by VahidNaderi** [comment]
IMO a modal is a little too much for this task, and what Wordpress folks did is nice but does not match current Orchard's overall design.

Maybe a check box as suggested by @sebastienros would be a good option too, something like this:
![orchardissue2](https://cloud.githubusercontent.com/assets/3517756/17239198/8b73d62a-5577-11e6-8024-7f7ac27c1fe9.jpg)

**Comment by sfmskywalker** [comment]
I agree. I like that the Publish Later checkbox and date time controls are now vertically stacked, looks cleaner, and that there is no longer two separate "Publish" buttons, making it less confusing.

**Comment by Skrypt** [comment]
let's "at least" for the new admin theme use a better looking datetime picker like this one : https://tarruda.github.io/bootstrap-datetimepicker/

**Comment by sebastienros** [comment]
@Skrypt priorities ...

**Comment by Skrypt** [comment]
I agree on that. Though it's not bad to keep those links somewhere. We never know !

**Comment by sebastienros** [comment]
Good point from @HermesSbicego-Laser, it's ambiguous what Save and Publish will do when the checkbox is checked. Will Publish publish now AND later, or just later? Then if just later, what is the difference with Save?
